components/User/UserRegistration/UserRegistration.js

```diff
@@ -116,10 +134,55 @@ class UserRegistration extends React.Component {
             return (usernameNotAllowed !== undefined) ? !usernameNotAllowed : true;
         });
 
-        $('.ui.form').form(validationRules);
+        $('.ui.form').not('.registrationmodalform').form(validationRules);
```

Comment:
Use react identifiers instead of searching for classes! Searching for classes caused problems before.